### DISCUSSION TITLE
CLion 由于找不到QCefview.dll无法继续执行代码

### DISCUSSION THREAD

**Discussion by ZQ-M** [CefView/QCefView#249] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88067052/237057977-8f390f3d-e205-405e-a86b-6db49bd6b2f6.png)
CLion+WINDOWS10
发行包 v1.07
CMakelist.txt包含了include并且target_link_libraries（QCefView）
官网快速开始代码
怎么解决这个问题？
尝试解决
重构Cmake-Debug-build后，会出现报错
![image](https://github.com/CefView/QCefView/assets/88067052/07c36cf8-64c7-4831-b6fb-3e5d1d55ea5c)
报错如下：include\ui_Amap.h(29): error C2061: 语法错误: 标识符“QCefView”

**Comment by tishion** [comment, marked as answer]
你这个属于自己的依赖问题，如果你是想把整个QCefview项目作为源码引入你的项目，你需要确认QCefview能够在你的clion环境下编译，并且给你自己的项目添加依赖。

**Comment by L-Super** [comment]
有没有可能，你的可执行文件路径下，没有此动态库导致的
